src/BaGet.UI/src/DisplayPackage/Dependents.tsx

```diff
@@ -40,6 +50,25 @@ class Dependents extends React.Component<IDependentsProps, IDependentsState> {
     }).catch((e) => console.log("Failed to load dependents.", e));
   }
 
+  private getDependentsMessage() {
+    const hits = this.state.totalHits ?? -1;
+    const packageId = this.props.packageId;
+
+    if(hits < 0) {
+      return ""
+    }
+
+    if(hits === 0) {
+      return `No packages depend on ${packageId}.`;
+    }
+
+    if(hits < 20) {
+      return `Showing ${hits} packages that depend on ${packageId}.`;
```

Comment:
This message shouldn't be plural if there's only a single result. What do you think of just always showing the `Showing the top 20 packages that depend` if there are 1 or more hits, even if there are more than 20?